foundationdb/examples/range.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 foundationdb-rs developers, https://github.com/bluejekyll/foundationdb-rs/graphs/contributors
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+extern crate foundationdb;
+extern crate foundationdb_sys;
+extern crate futures;
+extern crate rand;
+
+use foundationdb::keyselector::*;
+use foundationdb::*;
+
+use futures::future::*;
+use futures::stream::*;
+
+use error::FdbError;
+
+fn random_str(len: usize) -> String {
+    use rand::Rng;
+    let mut rng = rand::thread_rng();
+    rng.gen_ascii_chars().take(len).collect::<String>()
+}
+
+fn example_get_range() -> Box<Future<Item = (), Error = FdbError>> {
```

Comment:
for the Boxed futures, I bet we could come up with a Generic type so that we don't need to define so many. I'll take a look at this.